src/Tools/DynamoShapeManager/App.config

```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<configuration>
+    <appSettings>
+        <add key="productsWithASM" value="Revit,Civil,Robot Structural Analysis,FormIt"/>
```

Comment:
Should I add ALIAS here too ?
Or should we leave it to DEVs/Sandbox users to specify more products

Reply:
if this is intended for 2.13 my vote is don't add it. If it's just for 2.14 then go for it.